src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumnManager.java

```diff
@@ -113,6 +114,13 @@ protected State()
          return JsUtil.toStringArray(getNamesNative());
       }
 
+      public final native String getActiveColumn() /*-{
+         if (this.activeColumn)
```

Comment:
IMHO, a more idiomatic way of specifying this is JavaScript would be `this.activeColumn || ""`.

Reply:
Much nicer, thank you!!